README.md

```diff
@@ -1 +1,57 @@
-# psc-discrepancy-parser
\ No newline at end of file
+# psc-discrepancy-parser
+This AWS Lambda service processes batches of CSV-format survey results (each survey gathered discrepancies with a company's Person(s) of Significant Control (PSC)), transforming a survey (in the form of a CSV record) into JSON and sending it to CHIPS.
+
+The batches of survey results are sent by email to an address whose emails will automatically be put in an S3 bucket in a particular folder. Files arriving in that folder trigger this Lambda to run. The Lambda will attempt to read the file as an email containing a CSV attachment in a particular format, transforming each CSV record to a PscDiscrepancySurvey JSON object and sending it to CHIPS over an HTTP POST. If any of these steps fail, the file is moved to a failed-to-process folder. If all succeed, the file is moved to a successfully-processed folder.
```

Comment:
Can we specify the origin folder as `source` as well as change the names for "failed-to-process" and "successfully-processed" to `rejected` and `accepted` respectfully.

Reply:
Good point, will do.